box-ui/src/components/ts/Student/StudentMeeting.tsx

```diff
@@ -65,6 +65,9 @@ const StudentMeeting: FunctionComponent = () => {
     //Animation loading, waiting for new Image
     const [loading, setLoading] = useState<boolean>(false);
 
+    //websocket
+    const [ws1, setWs1] = useState<WebSocket>();
```

Comment:
```suggestion
    const [ws, setWs] = useState<WebSocket>();
```
we use only one socket